test/client/docs_test.rb

```diff
@@ -518,6 +518,19 @@
       assert docs
       assert_equal(%w[1 2], docs.map { |h| h['_id'] }.sort)
     end
+
+    it 'percolates' do
+      populate!
+
+      response = @index.register_percolator_query "1", { :query => { :match => { :author => "pea53" } } }
+      assert response["created"]
```

Comment:
For generic asserts, I like to add a message to describe what went wrong (the default message is "Assertion failed" or something equally useless).

```ruby
assert response["created"], "Couldn't create the percolator query"
```